Manon-Seb/tp-space-x
# [Front] Create capsules view

## Task
Create capsules view + header component

## Type of change

## Description
- Add the capsules component
- Add the app header component
- Update homepage content

## Testing scenario
- Launch docker: `docker-compose up`
- Launch the container: `docker exec space-x /bin/sh`
- Go to the url `localhost:4200`
- Go to the url `localhost:4200/capsules` to see the capsules list
